unstructured_ingest/v2/processes/connectors/couchbase.py

```diff
@@ -219,6 +219,7 @@ def run(self, **kwargs: Any) -> Generator[FileData, None, None]:
 
 
 class CouchbaseDownloaderConfig(DownloaderConfig):
+    id_name: str = Field(default="id", description="The name of the id field in the collection")
```

Comment:
@ryannikolaidis should this be id_name or maybe better id_field? We need to be able to identify which field is unique. Its not always "id"

Reply:
how about `collection_id`? particularly so as not to be confused with any other ids of content in couchbase.

Reply:
Will change it to `collection_id` for now, but like @potter-potter says, it might be confusing since this implies the specific id of a collection instead of a parameter name. Maybe `collection_id_key`?